src/app/containers/FrontPageSection/index.jsx

```diff
@@ -64,7 +69,12 @@ const FrontPageSection = ({ bar, group, sectionNumber }) => {
           ))}
         </StoryPromoUl>
       ) : (
-        <StoryPromo item={items[0]} script={script} topStory />
+        <StoryPromo
+          item={items[0]}
+          script={script}
+          topStory
+          lazyLoadImage={false}
```

Comment:
I think this needs to be `lazyLoadImage={sectionNumber !== 0}` instead of `lazyLoadImage={false}`. As it stands, any single item section, anywhere on the page, will have a non lazy loaded image

Reply:
Also just noticed, I think our logic for the top story is wrong too, as (i think) any single item section will use the top story design, which is wrong

Reply:
Yep, confirmed this locally.
![image](https://user-images.githubusercontent.com/11341355/60075918-383f5b00-971e-11e9-9594-84de96401813.png)

Can youy change it to
`isFirstSection = sectionNumber !== 0;`
```
topStory={isFirstSection}
lazyLoadImage={isFirstSection}
```
For the lone story please? Then update the PR description to cover the 2 logic changes

Reply:
@dr3 I can't seem to reproduce the above example.

Reply:
If you add
```
{
        "type": "special-reports",
        "title": "Special reports 2",
        "strapline": {
          "name": "Isi akụkọ"
        },
        "items": [
          {
            "headlines": {
              "headline": "Lee 'ọgbanje’ Igbo ji eme ọnụ"
            },
            "locators": {
              "assetUri": "/igbo/afirika-48384830",
              "cpsUrn": "urn:bbc:content:assetUri:igbo/afirika-48384830"
            },
            "summary": "Akwaeke Emezi ga-arụ dịka onye odide na onye nchịkọta ihe ejije si n'akwụkwọ o biputara a ga-egosi n'Amerịka.",
            "timestamp": 1558614890,
            "passport": {
              "category": {
                "categoryId": "http://www.bbc.co.uk/ontologies/applicationlogic-news/News",
                "categoryName": "News"
              },
              "campaigns": [
                {
                  "campaignId": "5a988e3139461b000e9dabf9",
                  "campaignName": "WS - Divert me"
                }
              ]
            },
            "cpsType": "STY",
            "indexImage": {
              "id": "107081116",
              "subType": "index",
              "href": "http://c.files.bbci.co.uk/EEBE/production/_107081116_54514074_183132359321509_3636644245028733967_n.jpg",
              "path": "/cpsprodpb/EEBE/production/_107081116_54514074_183132359321509_3636644245028733967_n.jpg",
              "height": 549,
              "width": 976,
              "altText": "Akwaeke Emezi",
              "copyrightHolder": "Instagram/AZEMEZI"
            },
            "options": {
              "isBreakingNews": false,
              "isFactCheck": false
            },
            "id": "urn:bbc:ares::asset:igbo/afirika-48384830",
            "type": "cps"
          }
        ],
        "semanticGroupName": "Special reports 2"
      }
```
To the groups array in `/Users/mcmild07/dev/simorgh/data/test/igbo/frontpage/index.json` and run `npm run dev` you will see it

Reply:
@dr3 can you push what you have in this branch. Even after adding that 👆 I can't see the difference

Reply:
Sure, if you go to `TopStoryBroken` and run `npm run dev`, view at <600px and the very bottom story is in the `top story` style